src/style.css

```
@@ -1 +1,17 @@
 /* styles go here */
+.form-group {
+  margin-bottom: 20px;
+}
+
+.form-field {
+  display: block;
+  margin: 10px 0;
+}
+
+.form-field:first-of-type {
+  margin-top: 0;
+}
+
+.form-field:last-of-type {
+  margin-bottom: 0;
+}
```

you can combine these selectors
```suggestion
.form-field:first-of-type,
.form-field:last-of-type {
  margin-bottom: 0;
}
```

---

I have margin-bottom for last-of-type and margin top for first-of-type